.codecov.yml

```diff
@@ -8,6 +8,7 @@ coverage:
 codecov:
   ci:
     - "travis-ci.com"
+    - "https://github.com/features/actions"
```

Comment:
I have never gotten this to work reliably within docker, which is why we pass the token directly in TravisCI

https://docs.codecov.io/docs/about-the-codecov-bash-uploader

https://github.com/rlworkgroup/garage/blob/master/Makefile#L176